createdisk-library.sh

```
@@ -164,12 +164,36 @@ function copy_additional_files {
+function remove_local_rpm_repo() {
+    local vm_ip=$1
+    ${SSH} core@${vm_ip} -- 'sudo rm -fr /home/core/packages'
+    ${SSH} core@${vm_ip} -- 'sudo rm -fr /etc/yum.repos.d/local.repo'
+}
+
 function install_additional_packages() {
     local vm_ip=$1
-    shift
-    if [[ ${BASE_OS} = "fedora-coreos" ]]; then
+    if [[ ${BUNDLE_TYPE} = "okd" ]]; then
+        create_local_rpm_repo ${vm_ip}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)